azure/scope/managedmachinepool.go

```diff
@@ -341,3 +338,25 @@ func (s *ManagedMachinePoolScope) GetCAPIMachinePoolAnnotation(key string) (succ
 	val, ok := s.MachinePool.Annotations[key]
 	return ok, val
 }
+
+// IsManagedAutoUpgrade checks if version is auto managed by AKS.
+func (s *ManagedMachinePoolScope) IsManagedAutoUpgrade() bool {
+	return isManagedVersionUpgrade(s.ControlPlane)
+}
+
+func getManagedMachinePoolVersion(managedControlPlane *infrav1.AzureManagedControlPlane,
+	machinePool *expv1.MachinePool) *string {
```

Comment:
Could keep that on the same line as above

Reply:
ack